1.8/usage/containerizers.md

```diff
@@ -0,0 +1,59 @@
+---
+post_title: Choosing and Using a Containerizer
+nav_title: Containerizers
+menu_order: 3.5
+---
+
+DC/OS supports both [the default Mesos containerizer and the Docker containerizer](http://mesos.apache.org/documentation/latest/containerizer/).
```

Comment:
I think the intro should be revised to discuss Mesos containerizers at a high level, and then down into the flavors supported. From the standpoint of a newbie "Docker containerizer" doesn't sound Mesos related at all. Is it okay to call it `Mesos Docker containerizer`? E.g. `DC/OS provides support for both the Mesos containerizer and the Mesos Docker containerizer.` 

Also, I think the value proposition is missing from the user perspective - why would they want to know about this feature in the first place? What problem is it solving?